atomic_reactor/plugins/post_tag_from_config.py

```diff
@@ -69,9 +70,9 @@ def get_component_name(self):
                                'Has the build succeeded?')
 
         try:
-            name = (self.workflow.built_image_inspect[INSPECT_CONFIG]
-                                                     ['Labels']
-                                                     ['name'])
+            labels = self.workflow.built_image_inspect[INSPECT_CONFIG]['Labels']
+            name_label = str(get_preferred_label_key(labels, "name"))
+            name = labels[name_label]
```

Comment:
No, not really. `get_preferred_label` will mask the `KeyError` if the label is missing